_includes/components/card-horizontal.html

```diff
@@ -0,0 +1,27 @@
+{% comment %}
+Card Horizontal
+- header {String}
+- description {String}
+- href {String (url)}
+- date {Date}
+- categories {Array}
+{% endcomment %}
+
+<div class="card-horizontal" data-filter-entry data-filter-terms="{{ include.categories | join: " " }}" data-active="true">
+  <div class="card-horizontal__image" seed="{{ include.date | date_to_string }}">
```

Comment:
The `seed` here is a placeholder for a future generative thumbnail, something like: https://codepen.io/nathanlong/pen/gOQbxdd